platforms/afu_top_ifc_db/ccip_std_afu_avalon_mm_legacy_wires.json

```diff
@@ -7,6 +7,7 @@
          "use the SystemVerilog interface described in the parent."
       ],
    "parent": "ccip_std_afu_avalon_mm",
+   "platform-shim-module-name": null,
```

Comment:
Do you see a reason why customers may continue to want legacy wires for memory? If not, does it make sense to remove it (perhaps in a different review)? Since we dont expose legacy wires for CCIP, the natural question is why only support different interface types for local memory alone?

Reply:
We need them for now for compatibility with your existing AFUs. I agree we may choose to remove them over time.

Reply:
right.